teuthology/lock/ops.py

```diff
@@ -297,3 +285,153 @@ def push_new_keys(keys_dict, reference):
                 log.error('failed to update %s!', hostname)
                 ret = 1
     return ret
+
+
+def reimage(ctx, machines, machine_type):
+    reimaged = dict()
+    with teuthology.parallel.parallel() as p:
+        for machine in machines:
+            log.info("Start node '%s' reimaging", machine)
+            update_nodes([machine], True)
+            p.spawn(teuthology.provision.reimage, ctx,
+                    machine, machine_type)
+            reimaged[machine] = machines[machine]
+            log.info("Node '%s' reimaging is complete", machine)
+    return reimaged
+
+
+def reimage_many(ctx, machines, machine_type):
```

Comment:
This is not `reimage_many`, giving sufix _many_ makes function to expect a number of machines for a given type. I suggest to rename it to something `reimage` or `reimage_machines`.

Reply:
Changed it to `reimage_machines`.